app/src/main/java/com/fs_sournary/weather/screen/main/fragment/current/CurrentViewModel.java

```diff
@@ -12,11 +24,56 @@
 
 public class CurrentViewModel extends BaseObservable implements BaseViewModel {
 
-    CurrentViewModel() {
+    public final ObservableBoolean isRefresh = new ObservableBoolean(false);
```

Comment:
pls use primitive variable and make it private

Reply:
Biến này em để theo dõi thuộc tính của current weather fragment và để public final